crates/tabby/src/chat.rs

```diff
@@ -1,10 +1,10 @@
-mod prompt;
+mod chat_prompt;
```

Comment:
Is it easier to simply refer to it as 'prompt,' considering it's already under the 'chat' submodule?

Reply:
This `ChatService` refers to the `prompt` module from `src/chat/prompt.rs`, the new `CompletionService` I added here refers to the `prompt` module from `src/completions/prompt.rs` (If we insist with the same name).

This is fine, I just renamed module names for better readability, so we know which service depends on which prompt module.